lib/matplotlib/collections.py

```diff
@@ -227,11 +224,9 @@ def _prepare_points(self):
                 xs = self.convert_xunits(xs)
                 ys = self.convert_yunits(ys)
                 paths.append(mpath.Path(np.column_stack([xs, ys]), path.codes))
-
-            if offsets.size > 0:
-                xs = self.convert_xunits(offsets[:, 0])
-                ys = self.convert_yunits(offsets[:, 1])
-                offsets = np.column_stack([xs, ys])
+            xs = self.convert_xunits(offsets[:, 0])
```

Comment:
Are we sure you can get rid of this check?

Reply:
I thought yes (it's not a big deal to convert some zero-sized arrays anyways) but then decided against checking that it doesn't cause any problem, so reverted the removal of the check.

Reply:
Sure - not saying that I think there *is* a problem, except, someone put the check in at some point for some reason.